### PR TITLE
Initial UI Updates

### PR DESCRIPTION
**Pull request by kusaasira**
- [x] Datatables Updates. These were switched to the latest suit for the UI.

- [x] Ajax loading for datatables.

- [x] Introduction of Modals instead of _create_ views

- [x] Icons introduced in place.

- [x] Nav-bar color and menus updated.

- [x] Front Page updated to suit UI colors.

- [x] Region, Group & Respondent Controller methods modified to suit updates on views.

### CODE REVIEW

**Comment by ashley7** [comment]
Okay

File: .env.production [hunk 1]
```diff
@@ -1,7 +1,7 @@
 APP_NAME=Agripoll
```
Comment (ashley7) [inline .env.production]: This is what you would change to Buzza Agripoll. The other olternative in the config, is only taken if this VARIABLE is not set. Other wise the AppName is still Agripoll

File: config/app.php [hunk 2]
```diff
@@ -14,7 +14,7 @@
     */
 
     // 'name' => env('APP_NAME', 'Agri Poll'),
-    'name' => 'Agri Poll',
+    'name' => 'Buuza Agripoll',
```
Comment (ashley7) [inline config/app.php]: You hard coded it here

Comment (kusaasira) [reply config/app.php]: The reason is that it is appearing on all the title pages and the App name is not used anywhere in the project FYI.

File: .env.production (hunk 1 above)
Comment (kusaasira) [reply .env.production]: Check my comment below.

**Comment by ashley7** [comment]
This is Okay

**Pull request merged**